multik-default/src/main/kotlin/org/jetbrains/kotlinx/multik/default/DefaultMath.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.kotlinx.multik.default
```

Comment:
This file has changed its location, now it is in `org.jetbrains.kotlinx.multik.default.math`